admin_guide/overcommit.adoc

```diff
@@ -42,6 +42,39 @@ the specified resource as is available with the lowest local priority. In low
 resource situations, containers that specify no resource requests are given the
 lowest quality of service.
 
+[[tune-buffer-chunk-limit]]
+=== Tune Buffer Chunk Limit
+
+If Fluentd logger is unable to keep up with a high number of logs, it will need
+to switch to file buffering to reduce memory usage and prevent data loss.
+
+The Fluentd `buffer_chunk_limit` is determined by the environment variable
+`buffer_size_limit`. The file buffer size per output is determined by the
+environment variable `file_buffer_limit`. The permanent volume size has to be
+larger than `file_buffer_limit` times number of output.
+
+On the Fluentd and Mux pods, permanent volume `/var/lib/fluentd` should be
+prepared by pvc or hostmount, for example. Then, that area is used for the file
+buffers.
+
+The `buffer_type` and `buffer_path` are configured in the Fluentd config files as
+follows,
+
+----
+$ egrep "buffer_type|buffer_path" *.conf
+es-copy-config.conf:       buffer_type file
+es-copy-config.conf:       buffer_path '/var/lib/fluentd/buffer-es-copy-config'
+es-ops-copy-config.conf:   buffer_type file
+es-ops-copy-config.conf:   buffer_path '/var/lib/fluentd/buffer-es-ops-copy-config'
+output-es-config.conf:     buffer_type file
+output-es-config.conf:     buffer_path '/var/lib/fluentd/buffer-output-es-config'
+output-es-ops-config.conf: buffer_type file
+output-es-ops-config.conf: buffer_path '/var/lib/fluentd/buffer-output-es-ops-config'
+----
+
+The Fluentd `buffer_queue_limit` is calculated as `file_buffer_limit` divided by
+`buffer_size_limit`.
```

Comment:
`file_buffer_limit` --> 'FILE_BUFFER_LIMIT'
`buffer_size_limit` --> 'FILE_SIZE_LIMIT'
Please note that 'buffer_queue_limit' remains lowercase since 'buffer_queue_limit' is not an environment variable but a directive in the fluentd's configuration.